docs/niswitch/examples.rst

```diff
@@ -1,6 +1,15 @@
 Examples
 ========
 
+.DS_Store
```

Comment:
You need to remove all non examples from this folder.

Reply:
If you merge #336 into your branch, files that don't end in .py won't be added as an example.

Reply:
will merge